quantization/awq/quant_autoawq.py

```diff
@@ -0,0 +1,52 @@
+import argparse
+import logging
+from awq import AutoAWQForCausalLM, models
```

Comment:
`awq` seems to be missing in `requirements.txt`.

Reply:
done